addons/event_sale/wizard/event_batch_configurator_line.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class EventConfiguratorLine(models.TransientModel):
+    """Event Configuration"""
+    _name = "event.editor.line"
+    _description = 'Edit Event Lines on Sales Confirmation'
+    _order = "id desc"
+
+    editor_id = fields.Many2one('event.event.configurator.batch')
+    product_id = fields.Many2one('product.product', string="Product", readonly=True)
+    event_ticket_id = fields.Many2one('event.event.ticket', string='Event Ticket')
+    event_id = fields.Many2one('event.event', string="Event")
+    order_id = fields.Many2many('sale.order', string="Sale order")
+    sale_order_line_id = fields.Many2many('sale.order.line', string="Sale Order Line")
+
+    def get_event_data(self):
+        self.ensure_one()
+        for line in self.sale_order_line_id:
+            if line.product_type == 'event' and not line.event_id.id:
+                line.write({
+                    'event_id': self.event_id.id,
+                    'event_ticket_id': self.event_ticket_id.id,
+                    'order_id': self.order_id.id,
+                    'display_type': line.display_type,
+                    'product_id' : line.product_id.id,
+                    'product_uom' : line.product_uom.id,
```

Comment:
why is this needed ? you rewrite the same value isn't it ? (display_type / product_id / product_uom)

is not necessary, I think the write can be done in batch